src/__tests__/store/message/reducer.test.ts

```diff
@@ -0,0 +1,176 @@
+import { messageActionTypes, IMessageState } from "../../../store/message/types";
+import { messageReducer } from "../../../store/message/reducer";
+
+const initialState: IMessageState = {
+    message: {
+        message: "hello",
+        type: "success",
+        displayType: "modal"
+    },
+    confirmation: {
+        title: "world",
+        text: "hi",
+        confirm: {
+            label: "cat",
+            action: "do",
+            data: "anytype"
+        },
+        dismiss: {
+            label: "dog",
+            action: "doit",
+            data: "anytype"
+        },
+    }
+};
+const testState : IMessageState = {
+    message: {
+        message: "hello1",
+        type: "error",
+        displayType: "toastr"
+    },
+    confirmation: {
+        title: "world1",
+        text: "hi1",
+        confirm: {
+            label: "tiger",
+            action: "doitagain",
+            data: 1
+        },
+        dismiss: {
+            label: "lion",
+            action: "do",
+            data: 2
+        },
+    }
+}
+
+
+describe('message reducer', () => {
+    it('should return the initial state', () => {
+        expect(messageReducer(initialState,  {
+            type: null,
+        })).toEqual(
+
+            initialState
+
+        )
+    })
+
+    it('should display message', () => {
+        expect(messageReducer(initialState, {
+            type: messageActionTypes.DISPLAY_MESSAGE,
+            payload: testState
+        })).toEqual(
+            {
```

Comment:
simplify this to use initialState instead of repeating the contents of the obj every time. this test case and all others.

Reply:
Already deleted